src/modules/claim/entities/roleClaim.entity.ts

```diff
@@ -12,6 +12,14 @@ export class RoleClaim implements IRoleClaim {
   static create(data: Partial<RoleClaim>): RoleClaim {
     const entity = new RoleClaim();
     const jwt = new JWT(new Keys());
+
+    if (
+      data?.requestCompletionRedirectUrl &&
+      !isURL(data.requestCompletionRedirectUrl, { require_tld: false })
```

Comment:
`require_tld` is for `localhost`, right? Maybe add comment about this?
```suggestion
      // https://github.com/validatorjs/validator.js/issues/754#issuecomment-346826183
      !isURL(data.requestCompletionRedirectUrl, { require_tld: false })
```